src/lib/widgets/episode/ui/episode-card.svelte

```diff
@@ -4,27 +4,25 @@
   import { LikeButton } from '$lib/features/like-episode';
   import { DownloadLink } from '$lib/features/download-episode';
   import { EpisodeCardShell } from '$lib/entities/episode';
-  import { play, pause, audio } from '$lib/entities/audio';
+  import { paused } from '$lib/entities/audio';
   import { addToListeningHistory } from '$lib/entities/audio/model/listening-history';
+  import { toggleGlobalPlayback } from '$lib/features/playback-controls';
 
   export let episode: Episode;
   export let podcastId: number | null = null;
 
-  $: playing = $audio?.src === episode.audioUrl && !$audio?.paused && !$audio?.ended;
-
-  const playAudio = () => {
-    if (playing) {
-      pause();
-    } else {
-      play(episode.audioUrl);
-      if (podcastId) addToListeningHistory(podcastId);
+  const togglePlaying = () => {
+    if ($paused && podcastId) {
+      addToListeningHistory(podcastId);
     }
+
+    toggleGlobalPlayback();
```

Comment:
issue: but now it won't know what audio to play.

suggestion: I think we should modify the `toggleGlobalPlayback` function to accept a URL